RegisterRoutes try/catch error is of type 'unknown'.

The `error` in the `try/catch` of the template for `koa` middleware is missing `any` type which causes TS (`v4.4.3`) to throw `error TS2571: Object is of type 'unknown'`.
https://github.com/lukeautry/tsoa/blob/9ea032adb7a0885305cb8d6793a9b660fee45592/packages/cli/src/routeGeneration/templates/koa.hbs#L85

Edit: Also spotted these which are likely to cause the same issue with `hapi` and `express`:
https://github.com/lukeautry/tsoa/blob/9ea032adb7a0885305cb8d6793a9b660fee45592/packages/cli/src/routeGeneration/templates/hapi.hbs#L104
https://github.com/lukeautry/tsoa/blob/9ea032adb7a0885305cb8d6793a9b660fee45592/packages/cli/src/routeGeneration/templates/express.hbs#L84

--- a/packages/cli/src/routeGeneration/templates/express.hbs
+++ b/packages/cli/src/routeGeneration/templates/express.hbs
@@ -81,7 +81,7 @@ export function RegisterRoutes(app: express.Router) {
             let validatedArgs: any[] = [];
             try {
                 validatedArgs = getValidatedArgs(args, request, response);
-            } catch (err) {
+            } catch (err: any) {
                 return next(err);
             }
 

--- a/packages/cli/src/routeGeneration/templates/hapi.hbs
+++ b/packages/cli/src/routeGeneration/templates/hapi.hbs
@@ -101,7 +101,7 @@ export function RegisterRoutes(server: any) {
                     let validatedArgs: any[] = [];
                     try {
                         validatedArgs = getValidatedArgs(args, request, h);
-                    } catch (error) {
+                    } catch (error: any) {
                         if (isBoom(error)) {
                             throw error;
                         }

--- a/packages/cli/src/routeGeneration/templates/koa.hbs
+++ b/packages/cli/src/routeGeneration/templates/koa.hbs
@@ -82,7 +82,7 @@ export function RegisterRoutes(router: KoaRouter) {
             let validatedArgs: any[] = [];
             try {
               validatedArgs = getValidatedArgs(args, context, next);
-            } catch (error) {
+            } catch (error: any) {
               context.status = error.status;
               context.throw(error.status, JSON.stringify({ fields: error.fields }));
             }

